assets/pages/bundles-landing/components/Bundles.jsx

```diff
@@ -380,7 +380,6 @@ function Bundles(props: PropTypes) {
     props.intCmp,
     props.campaign,
     props.otherQueryParams,
-    props.abTests,
```

Comment:
Was this added in the two test PRs?

Reply:
It was added in this one https://github.com/guardian/support-frontend/pull/349/files

Reply:
Ah, sorry, misread the diff, thanks!